cli/js/runtime_main.ts

```diff
@@ -95,11 +95,12 @@ export function bootstrapMainRuntime(): void {
     }
   });
 
-  const { args, cwd, noColor, pid, repl, unstableFlag } = runtime.start();
+  const { args, cwd, noColor, pid, repl, scriptUrl, unstableFlag } = runtime.start();
 
   Object.defineProperties(denoNs, {
     pid: readOnly(pid),
     noColor: readOnly(noColor),
+    scriptUrl: readOnly(scriptUrl),
```

Comment:
Could you make this a getter so that we can make a runtime check if --unstable was provided or not?